src/ophys_etl/transforms/play.py

```diff
@@ -0,0 +1,188 @@
+import h5py
```

Comment:
I don't think this file should be in the repo. If the functionality is not covered in a test, and it is important, it should be,

Reply:
Yep this was a quick file for testing something, thought I deleted it already.